spec/user_spec.rb

```diff
@@ -0,0 +1,16 @@
+require 'rails_helper'
+
+RSpec.describe User, type: :model do
+  subject { User.new(Name: 'Tom', Photo: 'https://unsplash.com/photos/F_-0BxGuVvo', Bio: 'Teacher from Mexico.') }
+  before { subject.save }
+
+  it 'name should be present' do
+    subject.Name = nil
+    expect(subject).to_not be_valid
+  end
+
+  it 'bio should be present' do
+    subject.Bio = nil
+    expect(subject).to_not be_valid
+  end
```

Comment:
- [ ] Great job working on the tests but I think you missed testing the method `recent_posts` in the User but since it is found in models so you should test it as shown in the requirements so kindly add test to the method ✨✅🔥

![image](https://user-images.githubusercontent.com/58553711/215096423-fbb8a9dd-0d87-4c7b-9e17-b994b5341eaf.png)